spec/services/atmosphere/billing_service_spec.rb

```diff
@@ -40,7 +45,9 @@
 
     it 'throws exception for malformed appliance' do
       appl = create(:appliance, fund: nil, appliance_set: wf, appliance_type: not_shareable_appl_type, appliance_configuration_instance: config_inst, virtual_machines: [not_shareable_vm1])
-      appl.update_column(:fund_id, nil)
+
+      appl.update_columns(fund_id: nil)
```

Comment:
Can you explain why you are nilling this value with callbacks skipped? If there is dedicated callback/validation which guarantee that appliance without fund cannot exist maybe this spec can be removed?